textattack/transformations/word_insertions/word_insertion_masked_lm.py

```diff
@@ -20,6 +20,9 @@ class WordInsertionMaskedLM(WordInsertion):
             you can skip this argument as the correct tokenizer can be infered from the name. However, if you're passing the actual model, you must
             provide a tokenizer.
         max_length (int): the max sequence length the masked language model is designed to work with. Default is 512.
+        window_size (int): The number of surrounding words to include when making top word prediction.
```

Comment:
I am Out of Loop here, why is this window size necessary?

Reply:
It's an option if we do not want to look at the entire text to make the prediction for one [MASK] token.